src/99-appendices/04-entity-table.md

```diff
@@ -10,23 +10,23 @@ file type are denoted. Entity formats indicate whether the value is alphanumeric
-| headshape<br>(meg)                                                                             | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |
-| markers<br>(meg)                                                                               | REQUIRED      | OPTIONAL      | OPTIONAL       | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |
-| photo<br>(meg/eeg/ieeg)                                                                        | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       |                 |        
-| electrodes<br>(eeg/ieeg)                                                                       | REQUIRED      | OPTIONAL      |                | OPTIONAL      |                          |                |                          |               |                        |                |                     |                       | OPTIONAL        |
-| events<br>(meg/eeg/ieeg)                                                                       | REQUIRED      | OPTIONAL      | REQUIRED       |               |                          |                |                          | OPTIONAL      |                        |                |                     |                       |                 |        
+| Entity                                                                                         | Subject       | Session       | Task           | Acquisition   | Contrast Enhancing Agent | Reconstruction | Phase-Encoding Direction | Run           | Corresponding modality | Echo           | Recording           | Processed (on device) | Space           | Split           |
+| :--------------------------------------------------------------------------------------------- | :------------ | :------------ | :------------- | :------------ | :----------------------- | :------------- | :----------------------- | :------------ | :--------------------- | :------------- | :------------------ | :-------------------- | :---------------| :-------------- |
+| Format                                                                                         | `sub-<label>` | `ses-<label>` | `task-<label>` | `acq-<label>` | `ce-<label>`             | `rec-<label>`  | `dir-<label>`            | `run-<index>` | `mod-<label>`          | `echo-<index>` | `recording-<label>` | `proc-<label>`        | `space-<label>` | `split-<label>` |
```

Comment:
shouldn't this be `split-<index>` instead?

Reply:
no, should be `index` ... if we go after the definition @yarikoptic wants to introduce in https://github.com/bids-standard/bids-specification/pull/152/files

> **`<index>`** - a numeric value, possibly prefixed with arbitrary number of
    0s for consistent indentation, e.g., it is `01` in `run-01` following
    `run-<index>` specification.

thanks @teonbrooks